src/Command/CycleDependencyProxy.php

```diff
@@ -43,10 +44,17 @@ public function getSchema(): SchemaInterface
     {
         return $this->container->get(SchemaInterface::class);
     }
+    /**
+     * @deprecated {@see getSchemaProvider}
```

Comment:
Can't we remove it right away? There's no release so no need to keep things deprected.